src/main/kotlin/org/jitsi/nlj/rtp/bandwidthestimation/GoogleCcEstimator.kt

```diff
@@ -97,8 +97,8 @@ class GoogleCcEstimator(diagnosticContext: DiagnosticContext, parentLogger: Logg
     }
 
     override fun getStats(now: Instant): StatisticsSnapshot = StatisticsSnapshot("GoogleCcEstimator", getCurrentBw(now)).apply {
-        addNumber("latestDelayEstimate", sendSideBandwidthEstimation.latestREMB)
-        addNumber("latestFractionLoss", sendSideBandwidthEstimation.latestFractionLoss)
+        addNumber("latestDelayBasedEstimate", sendSideBandwidthEstimation.latestREMB)
+        addNumber("latestFractionLoss", sendSideBandwidthEstimation.latestFractionLoss / 256.0)
```

Comment:
`latestFractionLossPct` ?

Reply:
Well, it already has "fraction" in the name, and it's out of 1, not out of 100 -- maybe `latestLossFraction`?

Reply:
Yup, good point...if anything this is making it more consistent with the existing name.  sgtm.